src/ResponseHeaders.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+namespace Jasny\HttpMessage;
+
+/**
+ * ServerRequest header methods
+ */
+class ResponseHeaders extends Headers
+{
+    
+    /**
+     * HTTP headers
+     *
+     * @var bool
+     */
+    protected $isStale;
+
+    /**
+     * Create header array from resived array in the Header дшые
+     *
+     * @param array $incomingArray
+     * @response Header class object
+     */
+    public function __construct($incomingArray = [])
```

Comment:
The constructor shouldn't take any arguments. Just creating a `ResposneHeaders` object, shouldn't change the global state.